.github/issue_commands.json

```diff
@@ -0,0 +1,18 @@
+[   
+    {
+      "type": "label",
+      "name": "datasource/TwinMaker",
+      "action": "addToProject",
```

Comment:
I'm not really sure when this is called/used. Could you explain that? 
So I added one entry for both existing entries with addToProject. Was that right?

Reply:
ex: I add the label "datasource/TwinMaker" to a ticket, I want it to get added to our kanban board, if I remove "datasource/TwinMaker" it would remove it from our board...not sure how much this really comes up in our workflow, but we tend to do this lol

Reply:
Just tried this out here https://github.com/grafana/grafana-iot-twinmaker-app/issues/159 (you can see I've added the label, saw it add to our board). I removed datasource/TwinMaker and pretty sure it's still stuck on our board

Reply:
Nevermind it's gone now!

Reply:
Nevermind I was wrong! It has **not** been removed from our project board.